docs/v1/use-cases.md

```diff
@@ -1,96 +1,143 @@
-that service.
-
-#### Updating a service broker
-
-Broker authors make changes to the services their brokers offer.  To refresh the
+Broker authors make changes to the services their brokers offer. To refresh the
 services a broker offers, the catalog should re-list the `/v2/catalog` endpoint.
 The catalog should apply the result of re-listing the broker to its internal
 representation of that broker's services:
 
-1.  New service present in the re-list results are added
-2.  Existing services are updated if a diff is present
-3.  Existing services missing from the re-list are deleted
+1. New service present in the re-list results are added
+2. Existing services are updated if a diff is present
+3. Existing services missing from the re-list are deleted
 
 TODO: spell out various update scenarios and how they affect end-users
 
-#### Delete a service broker
+### Delete a service broker
 
-There must be a way to delete brokers from the catalog.  In Cloud Foundry, it is
-possible to delete a broker and leave orphaned service instances.  We should
+There must be a way to delete brokers from the catalog. In Cloud Foundry, it is
+possible to delete a broker and leave orphaned service instances. We should
 evaluate where all broker deletes should:
 
-1.  Cascade down to the service instances for the broker
-2.  Leave orphaned service instances in the catalog
-3.  Fail if service instances still exist for the broker
-
-## Supporting multiple backend APIs
-
-The CF service broker API is under active development, leading to two
-possibilities that may both occur:
-
-1.  The `v2` API undergoes backward-compatible changes
-2.  There is a new `v3` API that is not backward-compatible
-
-The service catalog should be able to support new backward-compatible fields or
-a new backend API without a major rewrite.  This should be kept in mind when
-designing the architecture of the catalog.
-
+1. Cascade down to the service instances for the broker
+2. Leave orphaned service instances in the catalog
+3. Fail if service instances still exist for the broker
+
+## Catalog Publishing/Curation/Discovery
+
+* How are services identified: name, service name/id, plan name/id?
+* Who can see which services? (TODO: Include scope? Global/Namespaced)
+* Who can see which service instances? (TODO: Include scope? Global/Namespaced)
+* Which service instances are globally visible?
+* Catalog curation: which broker-provided services are globally visible?
+* Catalog curation: which namespaces can see which catalog services?
+* Does my catalog list not only my service types but also my instances?
+  * E.g., I have a dev cluster running on my local machine and I want to connect
+    my application with a database that exists in a test cluster. How do I find
+    that database instance?
+
+## Requesting Services (Consumer)
+
+* As a User, how do I ask for a new service from the Catalog?
+* As a User, how do I ask for access to an existing service?
```

Comment:
Not sure what "access to an existing service" means. Do you mean "how do I request a new instance of a service?" or do you mean "how do I bind my app to an existing service insance?" ?